test/integration/pretty-various-types/pretty-various-types.test.ts

```diff
@@ -0,0 +1,115 @@
+import { calculate } from '../../../src/calculation/Calculator';
+import { CalculationOptions, DetailedPopulationGroupResult } from '../../../src/types/Calculator';
+import { getJSONFixture, getGroupByIndex } from '../helpers/testHelpers';
+
+const CALCULATION_OPTIONS: CalculationOptions = {
+  measurementPeriodStart: '2022-01-01',
+  measurementPeriodEnd: '2022-12-31',
+  enableDebugOutput: true
```

Comment:
do we need this option?
```suggestion
  enableDebugOutput: true
```

Reply:
I guess not- it can be helpful when you're working with the feature directly, but probably doesn't need to be in the integration tests long term.

Reply:
We have it in a few of integration tests. I'll remove it from all of them!